foyer/tests/utils.py

```diff
@@ -3,8 +3,7 @@
 import numpy as np
 
 
-
-def atomtype(structure, forcefield):
+def atomtype(structure, forcefield, non_atomistic=False):
```

Comment:
Is this `non_atomistic` flag only required here because this renaming would normally happen in `Forcefield` and doesn't in the way we're doing the tests?